CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+## Publishing ppx package
+
+In order to publish `@ahrefs/bs-emotion-ppx` package, the following steps are needed:
+
+- Go to dev.azure.com, and in the page of the build that will be published ([example](https://dev.azure.com/igor0048/ahrefs/_build/results?buildId=4)), press the top right "Artifacts" button and download "Release" zip.
+- In the downloaded unzipped folder, replace the following line in `package.json`:
```

Comment:
@ulrikstrid Thanks a lot for the work you did in the publishing side! 🙂 

Do you know if there's a way to automate this process without having to change this line in resulting `package.json`? I guess using esy instead of bsb-native would simplify the process?

If you have any pointers, or existing ppx repos that have solved this issue, please let us know. Thanks!

Reply:
Hey, I've only used `esy` + `dune` for native development so I'm not sure what's up here. In our setup for `esy` we generate a `package.json` that gets bundled in the final package.

You can find our CI setup here: https://github.com/esy-ocaml/hello-reason. We build a binary for every platform and then in a final step we move them into separate folders, puts a `postinstall.js` that will relocate the correct binary. Then we generate a `package.json` into that folder.

Feel free to hit me up on Discord or continue discussion here 👍

Reply:
Ah, that makes sense.

> Then we generate a `package.json` into that folder.

I think this is the step missing in `bs-emotion-ppx`. Right now, the "development `package.json`" is directly copied in the resulting artifacts:

https://github.com/ahrefs/bs-emotion/blob/b64db1628e915fe91a42217fd890dc627fc21d45/.ci/bs-emotion-ppx/bundle.yml#L45

 so it contains the wrong `postinstall` script that has to be changed. I guess generating a new `package.json` from scratch would fix this issue.

I will noodle a bit with these ideas in another PR, thanks for your help!

Reply:
Just for reference: https://github.com/esy-ocaml/hello-reason/blob/master/.ci/pipelines-release.js#L6